modules/302-vertical-pod-autoscaler/images/updater/patches/README.md

```diff
@@ -0,0 +1,3 @@
+### daemonset-eviction.patch
+
+Supports DaemonSets in pods eviction controller allowing to control the eviction rate of DaemonSet pods.
```

Comment:
Are there any PRs to the upstream repo?

Reply:
we create upstream patch after approve this PR

Reply:
The time has come, let's open a PR.

Reply:
I will add a task for myself

Reply:
https://github.com/kubernetes/autoscaler/pull/4978